bundle/edu.gemini.itc.web/src/main/resources/servlets/web/ITCgsaoi.html

```diff
@@ -443,53 +383,52 @@
 				condition constraints</a> pages. </td>
 			</tr>
             <tr>
-                <td><b>Image quality:</b></td>
+                <td><b>Image Quality:</b></td>
                 <td><input name="ImageQuality" value="PERCENT_20" type="radio" /> 20%/Best</td>
                 <td><input name="ImageQuality" value="PERCENT_70" type="radio" checked /> 70%/Good</td>
                 <td><input name="ImageQuality" value="PERCENT_85" type="radio" /> 85%/Poor</td>
-                <td><input name="ImageQuality" value="ANY" type="radio" /> Any</td>
+                <td><input name="ImageQuality" value="ANY"        type="radio" /> Any</td>
             </tr>
             <tr>
-                <td><b>Sky transparency (cloud cover):</b></td>
+                <td><b>Cloud Cover:</b></td>
                 <td><input name="CloudCover" value="PERCENT_50" type="radio" checked /> 50%/Clear</td>
                 <td><input name="CloudCover" value="PERCENT_70" type="radio" /> 70%/Cirrus</td>
                 <td><input name="CloudCover" value="PERCENT_80" type="radio" /> 80%/Cloudy</td>
-                <td><input name="CloudCover" value="ANY" type="radio" /> Any</td>
+                <td><input name="CloudCover" value="ANY"        type="radio" /> Any</td>
             </tr>
             <tr>
-                <td><b>Sky transparency (water vapour):</b></td>
+                <td><b>Water vapor:</b></td>
```

Comment:
Water Vapor? (To be in line with rest of capitalizations.)

Reply:
Yes, thanks!